pallets/corporate-actions/src/lib.rs

```diff
@@ -479,7 +503,7 @@ decl_module! {
         ///
         /// ## Errors
         /// - `UnauthorizedAsAgent` if `origin` is not `ticker`'s sole CAA (owner is not necessarily the CAA).
-        #[weight = <T as Trait>::WeightInfo::set_did_withholding_tax()]
+        #[weight = <T as Trait>::WeightInfo::set_did_withholding_tax(1)]
```

Comment:
Constant as argument, we could remove it.

Reply:
See note above.